hosts

```diff
@@ -0,0 +1,11 @@
+[control]
+ansible ansible_host=10.42.0.2
+
+[web]
+node-4 ansible_host=10.42.0.4
```

Comment:
Looks like this didn't get changed back from a previous commit made in this PR.

Reply:
ip: "10.42.0.#{i+2}". 1 is for the gateway on the host, 2 is for the ansible guest.

Reply:
Hum. Looking at what's in master and in some of the decks node-1 starts at 10.42.0.6 though. Not what I'm see in 41af437.

https://raw.githubusercontent.com/ansible/lightbulb/master/Vagrantfile

@dfederlein: Sanity check in isle 5?

Reply:
I can't grasp what you want @timo, it's just an IP range. The loop starts with 1, add whatever you fancy.

Reply:
@bbaassssiiee: I just wanted to leave the IP addresses the way they are in the Vagrantfile currently in master as I linked to. So instead of `ip: "10.42.0.#{i+2}"` in the node definition, `ip: "10.42.0.#{i+5}"` (I think) would produce the same lab setup as exists today. Hope that helps clarify.